src/Rook.cpp

```diff
@@ -97,7 +97,8 @@ void Rook::compute_moves()
                 {
                     y += dy[ray];
                     x += dx[ray];
-                    this->moves_from[square][ray] |= (util::constants::ONE << (y * BOARD_SIZE + x));
+                    this->moves_from[square][ray] |=
+                        (util::constants::ONE << (y * BOARD_SIZE + x));
```

Comment:
We do have a `to_bitboard` function in `util::` but I think we need to reorganize things because `Util.h` is following the anti-pattern of being a miscellaneous toolbox that forces unnecessary recompilation of almost all other units.

Reply:
Opened https://github.com/zxul767/pawn/issues/24